src/rules/rule-non-nested-empty-line-before/README.md

```diff
@@ -132,3 +132,51 @@ a
 b
 {}
 ```
+
+## Optional options
+
+### `except: ["after-single-line-comment"]`
+
+Reverse the primary option if the rule is after a single line comment.
+
+For example, with `"always"`:
+
+The following patterns are considered warnings:
```

Comment:
👍  Done in https://github.com/stylelint/stylelint/pull/1543/commits/ec02cc3a2e0961d42c9be0633d4e2b4ce945c154